### DISCUSSION TITLE
does octo support creating and editing `discussion` thread?

### DISCUSSION THREAD

**Discussion by molleweide** [pwntester/octo.nvim#239] [Q&A] [answered]
Hi guys,

I just found this plugin and am super excited to get started with it.
I am wondering if `octo` supports creating and modifying discussion threads?

**Comment by pwntester** [comment]
Not currently, PRs are welcome though 😃

**Reply by molleweide** [reply]
Allright! If I can I will try to help!!!

**Reply by williambdean** [reply]
I don't think anything came of this, did it? @pwntester 
I am willing to take a stab at this

EDIT: Working on this in #918

**Comment by williambdean** [comment]
I've started working on #643

**Comment by williambdean** [comment, marked as answer]
I've begin work on this in #918 (and is the way that I am making this comment as well!). 

It adds ability to interact with discussion titles, bodies, and comments as well as marking something as answered / unanswered. More to come but likely in additional PRs (replies). See the PR for more details.

If you would like to try out, feel free to check out the branch and provide some feedback in the PR!
